src/sentry/api/endpoints/organization_invite_request_details.py

```diff
@@ -57,12 +46,8 @@ class OrganizationInviteRequestDetailsEndpoint(OrganizationEndpoint):
 
     def _get_member(self, organization, member_id):
         try:
-            return OrganizationMember.objects.get(
-                Q(invite_status=InviteStatus.REQUESTED_TO_BE_INVITED.value)
-                | Q(invite_status=InviteStatus.REQUESTED_TO_JOIN.value),
-                organization=organization,
-                user__isnull=True,
-                id=member_id,
+            return OrganizationMember.objects.get_member_invite_query(member_id).get(
+                organization=organization
```

Comment:
Does it make sense at all for organization to be passed as a parameter to `get_member_invite_query`?

Reply:
@wedamija I purposely left it out because I have a query where I don't have the organization. See: https://github.com/getsentry/sentry/pull/29708/files#diff-38bebb13f20409b4ea44e977a5c976b696c2e8f3e1ec02855b2581e5d422f4e5R378-R384

The alternative was to add the `organization_id` to the `callback_id` but I figured it was better to have a more minimalist approach for what data I need to send.